src/providers/fault-mgmt-providers/fault-action-provider.js

```diff
@@ -65,7 +65,7 @@ export default class FaultActionProvider {
                 value: 900000
```

Comment:
Could we please align the other options here with [the options in the Yamcs Web UI](https://github.com/yamcs/yamcs/blob/master/yamcs-web/src/main/webapp/projects/webapp/src/app/alarms/shelve-alarm-dialog/shelve-alarm-dialog.component.ts#L19)?
![Screenshot 2024-10-15 at 5 38 03 PM](https://github.com/user-attachments/assets/07329de9-985a-4231-9f7d-511d1d1f0a85)

Reply:
Done